client/src/App.js

```diff
@@ -1,6 +1,10 @@
 import React, { useState } from 'react';
 
+import {Route} from 'react-router-dom';
+
 import SavedList from './Movies/SavedList';
```

Comment:
Have you thought about trying to add the SavedList feature? It's a fun stretch problem. You can message me for hints if you need any :)